elliottlib/__init__.py

```diff
@@ -1,10 +1,11 @@
 import sys
+import pkg_resources
+
 if sys.version_info < (3, 6):
     sys.exit('Sorry, Python < 3.6 is not supported.')
 from .runtime import Runtime
 
 
 def version():
-    from os.path import abspath, dirname, join
-    filename = join(dirname(abspath(__file__)), 'VERSION')
-    return open(filename).read().strip()
+    stream = pkg_resources.resource_stream('elliottlib', 'VERSION')
```

Comment:
I think we also need the ability to run elliott locally. So if that file doesn't exist, we need to fallback to the `git describe` approach.

Reply:
fixed, good catch